messaging/tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/IncomingCompletionStageMethodVerification.java

```diff
@@ -57,24 +57,32 @@
  * }
  * </pre>
  */
-@Topics({
-    CompletionStageIncomingMethodVerification.VOID_METHOD,
-    CompletionStageIncomingMethodVerification.NON_PARALLEL,
-    CompletionStageIncomingMethodVerification.NON_VOID_METHOD,
-    CompletionStageIncomingMethodVerification.SYNC_FAILING,
-    CompletionStageIncomingMethodVerification.ASYNC_FAILING,
-    CompletionStageIncomingMethodVerification.WRAPPED_MESSAGE,
-    CompletionStageIncomingMethodVerification.OUTGOING_WRAPPED,
-    CompletionStageIncomingMethodVerification.INCOMING_OUTGOING_WRAPPED
-})
 @Test(groups = {"all", "incoming"})
-public class CompletionStageIncomingMethodVerification extends Arquillian {
+public class IncomingCompletionStageMethodVerification extends Arquillian {
 
     @Deployment
     public static JavaArchive createDeployment() {
         return TckDeploymentUtils.createDeployment(Bean.class);
     }
 
+    private static final String VOID_METHOD = "void-method";
```

Comment:
:question: why not an enum to this constants?

Reply:
I just tried doing that, but it doesn't work, the reason is the constant needs to be passed to the `@Incoming` and `@Outgoing` annotations on the beans being used for testing, and the parameter there is `String`. In Java, you can only pass a constant to an annotation, you can't for example pass `MyEnum.VOID_METHOD.name()` since that requires invoking a method.